src/hmem_synapseai.c

```diff
@@ -206,12 +206,12 @@ int synapseai_close_handle(void *ipc_ptr)
 
 int synapseai_host_register(void *ptr, size_t size)
 {
-	return -FI_ENOSYS;
+	return FI_SUCCESS;
 }
 
 int synapseai_host_unregister(void *ptr)
 {
-	return -FI_ENOSYS;
+	return FI_SUCCESS;
 }
```

Comment:
These are in the `else` of ``#if HAVE_SYNAPSEAI`, should they return -FI_ENOSYS to be consistent with an unsupported HMEM?

Reply:
Yes. I think you are right. Updated the PR